spec/lib/pre_assembly/bundle_spec.rb

```diff
@@ -20,13 +21,18 @@
       allow(RestClient).to receive(:post).with(a_string_matching(exp_workflow_svc_url), {}).and_return(instance_double(RestClient::Response, code: 200))
     end
     it 'runs images_jp2_tif cleanly using images_jp2_tif.yaml for options' do
-      # TODO: as we switch to using models (#172, #175, etc) this test should also switch
-      bc = context_from_proj('images_jp2_tif')
+      bc = context_from_proj_ar_model('images_jp2_tif')
```

Comment:
this begs the question -- why not just revise the method context_from_proj ?  Or maybe that's covered in a subsequent commit?

Reply:
my guess is that removal of `BundleContextTemporary` and associated cruft is left for a subsequent PR.  good to get rid of that stuff ASAP, but i'd be fine w/ doing that in another PR.

Reply:
in that case, in future, I would like to see TODO comments about it, and a ticket for it.

Reply:
#275

Reply:
> in that case, in future, I would like to see TODO comments about it, and a ticket for it

yeah, i think we're making allowances for messier PRs and more follow on work than we usual might, due to the tight timeline.  but agreed, makes reviewing and merging more of a pain than it usually is.